### PR TITLE
Fixed typos (thanks to Benjamin Weis)

### PR DESCRIPTION
**Pull request by PierreRaybaut**
reStructured Text --> reStructuredText
occurence --> occurrence
Occurrence --> Occurrence

### CODE REVIEW

**Comment by PierreRaybaut** [comment]
@ccordoba12 (and others): it may break an existing "installation" of Spyder based on latest revision (because of the change of name of some settings related to occurrence highlighting, this kind of thing). Anyway, it seems good to fix those shameful typographical errors ;-) (and I'm responsible for most of them...)

**Comment by PierreRaybaut** [comment]
By the way, this PR is required to be merged in order to add German translation to Spyder which is undoubtedly a good addition.

**Comment by PierreRaybaut** [comment]
@ccordoba12:
After some tests, it seems that it's quite harmless.
So unless you have second thoughts on this, I suggest merging this PR.

File: spyderlib/config/main.py [hunk 1]
```diff
@@ -719,7 +719,7 @@ def is_gtk_desktop():
               'zenburn/background':  "#3f3f3f",
               'zenburn/currentline': "#333333",
               'zenburn/currentcell': "#2c2c2c",
-              'zenburn/occurence':   "#7a738f",
+              'zenburn/occurrence':   "#7a738f",
```
Comment (ccordoba12) [inline spyderlib/config/main.py]: Changing option names requires a *major* bump in `CONF_VERSION` number (present below in this same file)

File: spyderlib/config/main.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
     (_("Patch and diff files"), ('.patch', '.diff', '.rej')),
     (_("Batch files"), ('.bat', '.cmd')),
     (_("Text files"), ('.txt',)),
-    (_("reStructured Text files"), ('.txt', '.rst')),
+    (_("reStructuredText files"), ('.txt', '.rst')),
```
Comment (ccordoba12) [inline spyderlib/config/main.py]: I don't like this change too much, but I have no problem with it :-)

**Comment by ccordoba12** [comment]
Other than the bump to `CONF_VERSION`, I think this is ready :-)

File: spyderlib/config/main.py (hunk 2 above)
Comment (PierreRaybaut) [inline spyderlib/config/main.py]: Honestly, I don't have any opinion on that... the contributor (Benjamin) seems right to point out this minor typo. If it allows Spyder to be translated to a new language, I'm OK with it :)

**Pull request merged**